docs/docs/03-tools/05-context.md

```diff
@@ -0,0 +1,64 @@
+# Context
+
+GPTScript supports context provider tools which can be used to provide additional context for tool execution. Output from the context tool gets prepended to the instruction of the current tool that uses the context tool.
+
+## Writing a Context Provider Tool
+
+Here is a simple example of a context provider tool that provides additional context to search tool:
```

Comment:
Note that they are just regular GPTScript tools, and any valid gptscript field can be used.

Reply:
This information is now included.